src/posts/introducing-inference-data/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Better Bayesian Workflows with Inference Data and Xarray'
```

Comment:
```suggestion
title: 'Better Bayesian Workflows with InferenceData and Xarray'
```

Are you referring to the class? or "inference data" as a type of data?

Reply:
The class